api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -626,6 +626,42 @@ public void getGlobalProperty_shouldReturnGlobalPropertyIfUserIsAllowedToView()
 		
 		assertNotNull(adminService.getGlobalProperty(property.getProperty()));
 	}
+
+	/**
+	 * @see org.openmrs.api.AdministrationService#getGlobalPropertyObject(java.lang.String)
+	 */
+	@Test
+	public void getGlobalPropertyObject_shouldFailIfUserHasNoPrivileges() {
+		executeDataSet(ADMIN_INITIAL_DATA_XML);
+		GlobalProperty property = getGlobalPropertyWithViewPrivilege();
+
+		// authenticate new user without privileges
+		Context.logout();
+		Context.authenticate("test_user", "test");
```

Comment:
The above method is deprecated. Can you use the recommended alternative?

Reply:
sure, I have changed it